package.json

```diff
@@ -1,36 +1,56 @@
 {
-  "name": "judge-d-ui",
-  "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-    "@types/jest": "24.0.18",
-    "@types/node": "12.7.2",
-    "@types/react": "16.9.2",
-    "@types/react-dom": "16.9.0",
-    "react": "^16.9.0",
-    "react-dom": "^16.9.0",
-    "react-scripts": "3.1.1",
-    "typescript": "3.5.3"
-  },
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
-  },
-  "eslintConfig": {
-    "extends": "react-app"
-  },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
-  }
+    "name": "@hltech/judge-d-ui",
+    "version": "0.1.0",
+    "private": true,
+    "dependencies": {
+        "d3": "^5.9.7",
+        "emotion": "^10.0.14",
+        "husky": "^2.3.0",
+        "prettier": "^1.17.1",
+        "pretty-quick": "^1.10.0",
+        "react": "^16.9.0",
+        "react-dom": "^16.9.0",
+        "react-scripts": "3.1.1",
+        "typescript": "3.5.3",
+        "semantic-ui-css": "^2.4.1",
+        "semantic-ui-react": "^0.87.3"
+    },
+    "devDependencies": {
+        "@types/node": "12.7.2",
+        "@types/react": "16.9.2",
+        "@types/react-dom": "16.9.0",
+        "@testing-library/jest-dom": "^4.0.0",
+        "@testing-library/react": "^9.1.1",
+        "@types/d3": "^5.7.2",
+        "@types/jest": "24.0.18",
+        "@types/react-test-renderer": "^16.9.0",
+        "react-test-renderer": "^16.9.0",
```

Comment:
redundant if you're using testing-library

Reply:
Done